README.rst

```diff
@@ -26,8 +31,11 @@ Licensed under Apache license, Version 2.0
 *WARNING:* Until this library reaches v1.0, the interfaces may change!
 
 Projects using widgetastic
---------------------------
-- ManageIQ integration_tests: https://github.com/ManageIQ/integration_tests
+---------------------------
+- ManageIQ [`integration_tests <https://github.com/ManageIQ/integration_tests>`_]
+- Satellite [`airgun <https://github.com/SatelliteQE/airgun>`_]
+- Cloud Services [`insights-qe <https://gitlab.cee.redhat.com/insights-qe>`_]
```

Comment:
I don't think putting an internal url makes sense here.

Reply:
sure restoring old style.

Reply:
@quarckster done

Reply:
@digitronik The issue is not in the style. This url https://gitlab.cee.redhat.com/insights-qe is not public and it should be removed.

Reply:
ahhhh sure thing